### DISCUSSION TITLE
Crouton and Mrchromebox script

### DISCUSSION THREAD

**Discussion by Dozer257** [dnschneid/crouton#4474] [Q&A]
Recently I have been doing a lot of testing with my Chromebook involving Linux. So the first time I decided to use crouton, I love it, in fact, I think it's way better than Crostini. I use crouton daily for schoolwork, chatting, and even playing Minecraft. Soon I realized some issues made Ubuntu more limited than a regular install of ubuntu, like not being able to install snap or using Ubuntu Software. Then I thought to myself, "I should scrap Chrome os because I don't use it anymore on this Chromebook, so I decide to use mrchromebox's script to install Ubuntu or Ubuntu/Debian based distros. It worked, but weirdly it was way laggier than installing Ubuntu using crouton. First I tried Linux Mint 20.1 (Cinnamon Desktop) it was really laggy and figured that it was probably the DE I was using. So I tried using Linux Mint 20.1 with Xfce. Still was laggy, I tried to install Manjaro, it didn't even install. I tried Ubuntu 20.04, didn't work either. Ubuntu 18.04 worked, but it was laggy as well. So I gave up on trying to install Linux standalone and went back to crouton. And Ubuntu wasn't laggy at all, for example, if I play Minecraft on crouton, I would get about 60 fps, but on a hard drive install of Linux, I only get 2-12 fps. This got me wondering:

"Why is a hard drive install of Ubuntu laggier than a dual boot of Ubuntu?"

Can someone please explain this?

**Comment by ghost** [comment]
Crouton sets everything up for you, with A fresh install there are a lot of things that you might need to setup your self (like drivers). What computer are you use?

**Reply by Dozer257** [reply]
HP Chromebook 11a G6 EE

**Reply by ghost** [reply]
Are you using the legacy boot script or UEFI, I've had better experiences with the UEFI one? If you do use the UEFI script you *may* need to reinstall Chrome OS, through Brunch or using a Chromium based OS like Cloud Ready.

**Reply by Dozer257** [reply]
I used legacy bc i cant open my Chromebook to remove wp screw, also are you saying uefi completely removes chrome os like the dev screen and recovery screen?

**Reply by ghost** [reply]
Sorry, for not replying. The UEFI script does remove the main part of the OS. It is possible to recover the Chromebook by using the script and putting the original firmware back on it, you will need a recovery media for this though (they aren't hard to make).

**Reply by Dozer257** [reply]
Ok, thanks.
